api/ds_client/ds_client.go

```diff
@@ -31,15 +31,33 @@ LICENSE
   OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
 
-package utils
+package ds_client
```

Comment:
Is the term "client" confusing since, isn't this supposed to be the server code?

This should be given a package comment too. All go packages should have a package comment starting with "Package <name> <description of what package does/ is used for".

Reply:
Client because it is the client in the interactions with Google's datastore service. I chose this name because it could not be datastore - that would clash. I think it is a bad name too but I haven't had any better ideas